### PR TITLE
OS-5: Changed ID to UUID for User

### PR DESCRIPTION
**Pull request by bartekkaminski**
https://openimis.atlassian.net/browse/OS-5

### CODE REVIEW

File: IMIS/FindUser.aspx [hunk 1]
```diff
@@ -268,7 +268,7 @@ In case of dispute arising out or in relation to the use of the program, it is s
         PagerSettings-FirstPageText = "First Page" PagerSettings-LastPageText = "Last Page" PagerSettings-Mode ="NumericFirstLast" 
         CssClass="mGrid" PageSize="15" DataKeyNames="UserID,LoginName,IsAssociated" EmptyDataText='<%$ Resources:Resource,L_NORECORDS %>'>
         <Columns>        
-            <asp:HyperLinkField DataNavigateUrlFields="UserID" 
+            <asp:HyperLinkField DataNavigateUrlFields="UserUUID"
```
Comment (dragos-dobre) [inline IMIS/FindUser.aspx]: The UserUUID field is not retrieved from the DB. You need to update also FindUser.aspx.vb. Verify also that the buttons' URL are updated with the UUID.

Comment (bartekkaminski) [reply IMIS/FindUser.aspx]: In **UsersDAL.vb** I introduced a change that the field is taken
`sSQL = " SELECT U.UserId, U.UserUUID, U.LanguageID, U.LastName, U.OtherNames, U.Phone, U.LoginName, U.RoleId, U.HFID, U.ValidityFrom, U.ValidityTo, U.LegacyId, U.AuditUserId, U.EmailId, U.IsAssociated"`

Comment (dragos-dobre) [reply IMIS/FindUser.aspx]: The FindUser works. I have an error when deleting an User:

```
User Control Panel : Admin : Conversion from string "c956c07c-df1b-4995-8239-57a45cd8" to type 'Integer' is not valid.
```

Comment (kmadej) [reply IMIS/FindUser.aspx]: @dragos-dobre thanks for feedback. @bartekkaminski  please verify once again and fix issues

**Comment by bartekkaminski** [comment]
I close the Pull Request due to the new one created

**Pull request closed without merging**